### PR TITLE
Add ProductSalesAnomalyDetection tutorial

### PR DESCRIPTION
**Pull request by JRAlexander**
[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sales-anomaly-detection?branch=pr-en-us-12131)

[Related Code Sample](https://github.com/dotnet/samples/pull/830)

[#AB1450810](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1450810)

### CODE REVIEW

File: docs/machine-learning/tutorials/sales-anomaly-detection.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+
+`Change points` are persistent changes in a time series event stream distribution of values, like level changes and trends. These persistent changes last much longer than `spikes` and could indicate catastrophic event(s). `Change points` are not usually visible to the naked eye, but can be detected in your data using approaches such as in the following method.  The following image is an example of a change point detection:
+
+![ChangePointDetection](./media/sales-anomaly-detection/ChangePointDetection.png)
+
+### Create the DetectChangepoint() method
+
+Add the following call to the `DetectChangepoint()`method as the next line of code in the `Main()` method:
+
+[!code-csharp[CallDetectChangepoint](~/samples/machine-learning/tutorials/ProductSalesAnomalyDetection/Program.cs#CallDetectChangepoint]
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/sales-anomaly-detection.md]: Check this code snippet. Might be broken.

**Review by luisquintanilla — CHANGES_REQUESTED** [review]
Nicely done. Minor changes

File: docs/machine-learning/tutorials/sales-anomaly-detection.md (hunk 1 above)
Comment (JRAlexander) [reply docs/machine-learning/tutorials/sales-anomaly-detection.md]: Yup. Missing an ending parenthesis. Good catch!

**Comment by JRAlexander** [comment]
Closing and reopening

**Review by luisquintanilla — APPROVED** [review]
Looks good.

**Pull request merged**